### PR TITLE
[12.0][FIX]hotel_reservation:Fixed multi company issue. 

### LINKED ISSUE #82: [12.0] Multi Company
Multi-company and unchecked "Common Product Catalog" in general settings.

when the user will make a record line in the "Reservations" menu at another company (for example hotel 2 and hotel 3),

Error: The requested operation could not be completed due to security restrictions. Please contact your system administrator.

(Document type: Warehouse, Operations: read) - (Note: [1], User: 8)

but the error does not apply to the main company,

and if we checked "Common Product Catalog" the user can access and make a line record via the "Reservations Summary" menu but with a dashboard display that shows all rooms(blunder) of the entire company (Hotel 1, Hotel 2, Hotel 3).

I have set permissions on the warehouse through the front-end, changing access rights through back-end security.csv
but errors related to the warehouse where the readonly operation still appears

Need help here.

### PR DESCRIPTION
**Pull request by dhruvitaSerpentCS**
Fixed #82

### CODE REVIEW

**Comment by dhruvitaSerpentCS** [comment]
@JayVora-SerpentCS  Can you please review pr and merge it?
Thanks.

File: hotel/models/hotel.py [hunk 1]
```diff
@@ -628,7 +628,7 @@ def onchange_partner_id(self):
                 self.partner_invoice_id = partner_rec.id
                 self.partner_shipping_id = partner_rec.id
                 self.pricelist_id = partner_rec.property_product_pricelist.id
-                raise ValidationError(_('Not Any Order For  %s ' % (partner_rec.name)))
+                raise ValidationError(_("Not Any Order For  %s " % (partner_rec.name)))
```
Comment (JayVora-SerpentCS) [inline hotel/models/hotel.py]: Can you edit as:
"No Order found for " ?

File: hotel/models/hotel.py [hunk 2]
```diff
@@ -1051,7 +1051,7 @@ def on_change_checkout(self):
         if not self.ser_checkout_date:
             self.ser_checkout_date = time_a
         if self.ser_checkout_date < self.ser_checkin_date:
-            raise ValidationError(_('Checkout must be greater or equal checkin date'))
+            raise ValidationError(_("Checkout must be greater or equal checkin date"))
```
Comment (JayVora-SerpentCS) [inline hotel/models/hotel.py]: Add an '!' at the end

File: hotel_reservation/__manifest__.py [hunk 3]
```diff
@@ -13,6 +13,7 @@
     "demo": ["data/hotel_reservation_data.xml"],
     "data": [
         "security/ir.model.access.csv",
+        "security/reservation_security.xml",
```
Comment (JayVora-SerpentCS) [inline hotel_reservation/__manifest__.py:15]: Add XML files first to render,  because CSV file will need the group existent before.

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

File: hotel/models/hotel.py (hunk 2 above)
Comment (dhruvitaSerpentCS) [reply hotel/models/hotel.py]: Done.

File: hotel_reservation/__manifest__.py (hunk 3 above)
Comment (dhruvitaSerpentCS) [reply hotel_reservation/__manifest__.py:15]: done

File: .flake8 [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [flake8]
 max-line-length = 79
-max-complexity = 16
+max-complexity = 21
```
Comment (JayVora-SerpentCS) [inline .flake8:3]: reason for this?

Comment (dhruvitaSerpentCS) [reply .flake8:3]: This change was done for travis fixing.

**Pull request merged**